### PR TITLE
[Feature] CTAS support insert data

### LINKED ISSUE #9143: [Enhancement] Let "create table as ... select" statement could insert data which query from select statement
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-doris/issues?q=is%3Aissue) and found no similar issues.

### Description

 Now in  Doris, the statement "create table as ... select" only create table and can not insert data, it is not comply with the statement's real semantic logic.

### Solution

_No response_

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by stalary**
# Proposed changes

Issue Number: close https://github.com/apache/incubator-doris/issues/9143

## Problem Summary:

Describe the overview of changes.

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
2. Has unit tests been added: (Yes/No/No Need)
3. Has document been added or modified: (Yes/No/No Need)
4. Does it need to update dependencies: (Yes/No)
5. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

CTAS support insert data, if insert error, drop table

### CODE REVIEW

File: fe/fe-core/src/main/java/org/apache/doris/analysis/CreateTableAsSelectStmt.java [hunk 1]
```diff
@@ -62,17 +73,6 @@ public void analyze(Analyzer analyzer) throws UserException {
...
-        return createTableStmt;
-    }
-    
-    public List<String> getColumnNames() {
-        return columnNames;
-    }
-    
-    public QueryStmt getQueryStmt() {
-        return queryStmt;
+        // Cannot analyze insertStmt because the table has not been created yet
```
[the remark this answers is not in the document]
Comment (stalary) [reply fe/fe-core/src/main/java/org/apache/doris/analysis/CreateTableAsSelectStmt.java:76]: > i think make this comment as a method function should be better

sorry, At present still under development, has not run, in the debugging stage

**Comment by stalary** [comment]
The current verification results are as follows
```sql
mysql> CREATE TABLE `table1` ( `id`   int          NOT NULL, `col1` varchar(255) NOT NULL, `col2` varchar(255) NOT NULL) ENGINE = OLAP UNIQUE KEY(`id`) COMMENT "测试表1" DISTRIBUTED BY HASH(`id`) BUCKETS 10
    -> PROPERTIES (
    -> "replication_num" = "1",
    -> "in_memory" = "false",
    -> "storage_format" = "V2"
    -> );
Query OK, 0 rows affected (0.12 sec)
```

```sql
mysql> insert into table1
    -> values (1, 'col1_1', 'col2_1'),
    ->        (2, 'col1_2', 'col2_2'),
    ->        (3, 'col1_3', 'col2_3'),
    ->        (4, 'col1_4', 'col2_4'),
    ->        (5, 'col1_5', 'col2_5');
Query OK, 5 rows affected (0.09 sec)
{'label':'insert_c02d578bd5434d53-b69baa4bfeb08377', 'status':'VISIBLE', 'txnId':'1003'}
```

```sql
mysql> create table table_ctas PROPERTIES ("replication_num" = "1") as select * from table1;
Query OK, 5 rows affected (0.21 sec)
{'label':'insert_e5cb0d80b2d0407f-9d659c4f7c885751', 'status':'VISIBLE', 'txnId':'1004'}
```

```sql
mysql> select *
    -> from table_ctas;
+------+--------+--------+
| id   | col1   | col2   |
+------+--------+--------+
|    3 | col1_3 | col2_3 |
|    2 | col1_2 | col2_2 |
|    5 | col1_5 | col2_5 |
|    4 | col1_4 | col2_4 |
|    1 | col1_1 | col2_1 |
+------+--------+--------+
5 rows in set (0.02 sec)
```

**Comment by stalary** [comment]
@morrySnow @lide-reed PTAL

**Comment by morrySnow** [comment]
please rebase to newest master code for passing FE UT

**Comment by stalary** [comment]
> please rebase to newest master code for passing FE UT

done

**Comment by stalary** [comment]
> please rebase to newest master code for passing FE UT

import has different from develop doc, and other I had modify,

**Comment by morrySnow** [comment]
> 

please follow the checker result, the document will be modified to fit checker

**Review by lide-reed — APPROVED** [review]
LGTM

**Pull request merged**